R/PipeOpFDACor.R

```diff
@@ -0,0 +1,76 @@
+#' @title Cross-Correlation of Functional Data
+#' @name mlr_pipeops_fda.cor
+#'
+#' @description
+#' Calculates the cross-correlation between two functional vectors using [tf::tf_crosscor()].
+#'
+#' @section Parameters:
+#' The parameters are the parameters inherited from [`PipeOpTaskPreprocSimple`], as well as the following
+#' parameters:
+#' * `arg` :: `numeric()` \cr
+#' Grid to use for the cross-correlation.
+#'
+#' @export
+#' @examples
+#' library(mlr3pipelines)
```

Comment:
maybe we want mlr3pipelines in Depends? mlr3da is completely unususable without it

Reply:
would agree, then we could also skip the library calls for mlr3pipelines in the examples etc.